scripts/upload-screenshots.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const PATH_TO_KEY = process.env.PATH_TO_KEY;
+const BUCKET_NAME = process.env.BUCKET_NAME;
+const COMMIT_HASH = process.env.COMMIT_HASH;
+const DIR = './test/screenshot';
+
+const Storage = require('@google-cloud/storage');
+const glob = require('glob');
+const path = require('path');
+const fs = require('fs');
+
+const storage = new Storage({
+  keyFilename: PATH_TO_KEY,
+});
+
+const bucket = storage.bucket(BUCKET_NAME);
+
+const screenshots = glob.sync(`${DIR}/**/*.png`);
+
+screenshots.forEach((fname) => {
+  const fileName = path.resolve(fname);
+  const bucketFileName = fname.replace(DIR, COMMIT_HASH);
```

Comment:
lets use PR instead of commit hash

Im worried that if we upload per commit to GCP, then we will end up storing a lot of files. At least with PR number you override the screenshots as you make changes within a PR.

And look into setting up a cron job in GCP to delete all the files in the screenshot-image-captures bucket.